package.json

```diff
@@ -28,30 +32,32 @@
     "access": "restricted"
   },
   "dependencies": {
-    "eslint-config-prettier": "^3.1.0",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-config-standard-jsx": "^6.0.2",
+    "@typescript-eslint/eslint-plugin": "2.x",
+    "@typescript-eslint/parser": "2.x",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-react-app": "^5.2.1",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-config-standard-jsx": "^8.1.0",
     "eslint-plugin-filenames": "^1.3.2",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^22.6.4",
-    "eslint-plugin-jsx-a11y": "^6.1.2",
-    "eslint-plugin-node": "^8.0.0",
-    "eslint-plugin-promise": "^4.0.1",
-    "eslint-plugin-react": "^7.11.1",
-    "eslint-plugin-react-hooks": "^1.6.0",
-    "eslint-plugin-react-perf": "^2.0.9",
-    "eslint-plugin-standard": "^4.0.0"
+    "eslint-plugin-flowtype": "4.x",
```

Comment:
Is this used/set in rules?

Reply:
I don't have it in rules, maybe should include. but since it's a peer dependency of eslint-config-react-app, it'll throw warnings and I don't like warnings :D